analyses/NE_cd_2010/doc_NE_cd_2010.md

```diff
@@ -0,0 +1,28 @@
+# 2010 Nebraska Congressional Districts
+
+## Redistricting requirements
+In Nebraska, districts must, under a [legislative resolution](https://nebraskalegislature.gov/FloorDocs/107/PDF/Intro/LR134.pdf):
+
+1. be contiguous
+1. have equal populations (specifically, within 0.5% of equality)
+1. be geographically compact
+1. preserve county and municipality boundaries as much as possible
+1. preserve the cores of prior districts
+1. not be drawn using partisan information
+
+
+### Interpretation of requirements
+We enforce a maximum population deviation of 0.5%.
+We apply a county constraint.
+We preprocess the map to ensure the cores of prior districts are preserved, as described below.
+
+## Data Sources
+Data for Nebraska comes from the ALARM Project's [Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+To preserve the cores of prior districts, we merge all precincts which are more than two precincts away from a district border under the 2000 plan.
```

Comment:
let's update this to 'one precinct'

Reply:
So do we want to go for one precinct boundaries instead of the current two?

Reply:
Oh, sorry, so the summary pictures you just showed are for `boundary=2`?  
Then nevermind, we're good to go.